.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
 *.log
-dist/
```

Comment:
This is debatable.

Reply:
This change was introduced only for the sake of my use case, so I can use my fork as dependency. Will of course be removed once we decide to merge.